Dockerfile

```diff
@@ -118,4 +159,5 @@ COPY --from=debian /usr/lib/${LIB_DIR_PREFIX}-linux-gnu/libblkid.so.1 \
 # Copy NVME support required script and rules into distroless base.
 COPY deploy/kubernetes/udev/google_nvme_id /lib/udev_containerized/google_nvme_id
 
-ENTRYPOINT ["/gce-pd-csi-driver"]
+COPY --from=builder /go/src/sigs.k8s.io/gcp-compute-persistent-disk-csi-driver/initialize-driver.sh /initialize-driver.sh
```

Comment:
This does not seem what we added? And it should consistent with master?

Reply:
Oh nvm I saw it bellow, somehow when I searched the keywords on the page it only recognizes the words in this file.